snuba/web/delete_query.py

```diff
@@ -38,11 +39,16 @@ class DeletesNotEnabledError(Exception):
     pass
 
 
+class TooManyOngoingMutationsError(Exception):
+    pass
+
+
 @with_span()
 def delete_from_storage(
     storage: WritableTableStorage,
     columns: Dict[str, list[Any]],
     attribution_info: Mapping[str, Any],
+    max_ongoing_mutations: int = 5,
```

Comment:
this number should be a runtime config option

Reply:
but can still use 5 as the default for the config